src/Entity.php

```diff
@@ -1733,58 +1710,41 @@ public static function showNotificationOptions(Entity $entity)
-            'use_certificates_alert' => $fn_get_specific_inheritance_label('use_certificates_alert'),
-            'send_certificates_alert_before_delay' => $fn_get_specific_inheritance_label('send_certificates_alert_before_delay'),
-            'certificates_alert_repeat_interval' => $fn_get_specific_inheritance_label('certificates_alert_repeat_interval'),
-            'use_reservations_alert' => $fn_get_specific_inheritance_label('use_reservations_alert'),
-            'notclosed_delay' => $fn_get_specific_inheritance_label('notclosed_delay'),
-            'use_domains_alert' => $fn_get_specific_inheritance_label('use_domains_alert'),
-            'send_domains_alert_close_expiries_delay' => $fn_get_specific_inheritance_label('send_domains_alert_close_expiries_delay'),
-            'send_domains_alert_expired_delay' => $fn_get_specific_inheritance_label('send_domains_alert_expired_delay'),
-            'approval_reminder_repeat_interval' => $fn_get_specific_inheritance_label('approval_reminder_repeat_interval'),
+            'mailing_signature' => $entity->getInheritedValueBadge('mailing_signature', inherit_parent_value: null),
+            'url_base' => $entity->getInheritedValueBadge('url_base', inherit_parent_value: null),
+            'cartridges_alert_repeat' => $entity->getInheritedValueBadge('cartridges_alert_repeat'),
+            'default_cartridges_alarm_threshold' => $entity->getInheritedValueBadge('default_cartridges_alarm_threshold'),
+            'consumables_alert_repeat' => $entity->getInheritedValueBadge('consumables_alert_repeat'),
+            'default_consumables_alarm_threshold' => $entity->getInheritedValueBadge('default_consumables_alarm_threshold'),
+            'use_contracts_alert' => $entity->getInheritedValueBadge('use_contracts_alert'),
+            'default_contract_alert' => $entity->getInheritedValueBadge('default_contract_alert'),
+            'send_contracts_alert_before_delay' => $entity->getInheritedValueBadge('send_contracts_alert_before_delay'),
+            'use_infocoms_alert' => $entity->getInheritedValueBadge('use_infocoms_alert'),
+            'default_infocom_alert' => $entity->getInheritedValueBadge('default_infocom_alert'),
+            'send_infocoms_alert_before_delay' => $entity->getInheritedValueBadge('send_infocoms_alert_before_delay'),
+            'use_licenses_alert' => $entity->getInheritedValueBadge('use_licenses_alert'),
+            'send_licenses_alert_before_delay' => $entity->getInheritedValueBadge('send_licenses_alert_before_delay'),
+            'use_certificates_alert' => $entity->getInheritedValueBadge('use_certificates_alert'),
+            'send_certificates_alert_before_delay' => $entity->getInheritedValueBadge('send_certificates_alert_before_delay'),
+            'certificates_alert_repeat_interval' => $entity->getInheritedValueBadge('certificates_alert_repeat_interval'),
```

Comment:
The inherited value is not converted into a human readable label.
![image](https://github.com/glpi-project/glpi/assets/33253653/f200e3a1-d4ee-4bb7-880b-2a04e740f282)